web/themes/contrib/civictheme/civictheme.post_update.php

```diff
@@ -619,3 +622,140 @@ function civictheme_post_update_enable_focal_point_configurations_2(): void {
+      '@theme_name' => $theme->getName(),
+      '@blocks_ids' => implode(', ', $updated_block_ids),
+      '@region_from' => $region_from,
+      '@region_to' => $region_to,
+    ]));
+  }
+
+  return (string) (new TranslatableMarkup('No blocks were moved.'));
+}
+
+/**
+ * Enables "civictheme_three_columns" layout for the Page/Event content type.
+ *
+ * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ */
+function civictheme_post_update_enable_three_column_layout(): string {
+  $outdated_layouts = [
+    'civictheme_one_column',
+    'civictheme_one_column_contained',
+  ];
+
+  $messages = [];
+
+  $entity_displays = LayoutBuilderEntityViewDisplay::loadMultiple();
+
+  $updated_entity_displays = [];
+  foreach ($entity_displays as $entity_display) {
+    if (!$entity_display->isLayoutBuilderEnabled()) {
+      continue;
+    }
+
+    // Updated 'allowed layouts' settings.
+    $entity_view_mode_restriction = $entity_display->getThirdPartySetting('layout_builder_restrictions', 'entity_view_mode_restriction');
+    if (!empty($entity_view_mode_restriction['allowed_layouts'])) {
+      $allowed_layouts = $entity_view_mode_restriction['allowed_layouts'];
+
+      foreach ($allowed_layouts as $layout_name) {
+        if (in_array($layout_name, $outdated_layouts)) {
+          unset($allowed_layouts[$layout_name]);
+        }
+      }
+
+      if (count($entity_view_mode_restriction['allowed_layouts']) != count($allowed_layouts)) {
+        $allowed_layouts[] = 'civictheme_three_columns';
+
+        $entity_view_mode_restriction['allowed_layouts'] = array_values($allowed_layouts);
+        $entity_display->setThirdPartySetting('layout_builder_restrictions', 'entity_view_mode_restriction', $entity_view_mode_restriction);
+
+        $updated_entity_displays[$entity_display->id()] = $entity_display->id();
+      }
+    }
+
+    // Replace layouts in sections.
+    /** @var \Drupal\layout_builder\Section[] $layout_builder_sections */
+    $layout_builder_sections = $entity_display->getThirdPartySetting('layout_builder', 'sections');
+    if (!empty($layout_builder_sections)) {
+      foreach ($layout_builder_sections as $index => $section) {
+        $layout_name = $section->getLayoutId();
+
+        if (in_array($layout_name, $outdated_layouts)) {
+          $section_as_array = $section->toArray();
+
+          $section_as_array['layout_id'] = 'civictheme_three_columns';
+          $section_as_array['layout_settings']['label'] = 'CivicTheme Three Columns';
+          $section_as_array['layout_settings']['is_contained'] = ($layout_name === 'civictheme_one_column_contained');
+
+          // Move all components to 'main' region because three column use
+          // 'main' region, not 'content' region as one column.
+          foreach ($section_as_array['components'] as &$component) {
+            if ($component['region'] === 'content') {
+              $component['region'] = 'main';
+            }
+          }
```

Comment:
Additonally add 
if ($component['configuration']['id'] === 'extra_field_block:node:civictheme_page:content_moderation_control') {
              $component['weight'] = -1;
            }
            so that moderation controls appear on the top